### PR TITLE
feat(pksd): add redis-server to docker dev

### PR DESCRIPTION
**Pull request by kgann**
Download and install Redis 6.2.1 for Clojure development.

### CODE REVIEW

**Comment by iku000888** [comment]
I guess I am curious of the end picture. 
Why manually add to the clojure-dev image over a ready to go image on the internet?

**Comment by kgann** [comment]
> I guess I am curious of the end picture.
> Why manually add to the clojure-dev image over a ready to go image on the internet?

We want to run a specific version of Redis in integration tests. Each test is isolated. If we ran Redis standalone (outside of integration-tests JVM) then we would need to introduce a way to clean the DB between tests and _could_ lose the ability to run in parallel. I did this after an ask From Kenji but didn't consider running the standalone image in `drone.yml`. Does that answer your question?

**Comment by iku000888** [comment]
:+1: 

Only had running in k8s in mind. Makes sense if this is a requirement for running the embedded redis in process.

**Comment by kgann** [comment]
> 👍
> 
> Only had running in k8s in mind. Makes sense if this is a requirement for running the embedded redis in process.

Ahh, Redis in k8s is an image already. https://github.com/parkside-securities/monorepo/tree/master/k8s/dev-base/redis

**Review by k2n — APPROVED** [review]
LGTM, thx!

File: Dockerfile [hunk 1]
```diff
@@ -37,6 +37,17 @@ RUN curl -sL https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/graphviz.tar
     ./configure && make && make install && \
     cd - && rm -rf graphviz*
 
+ENV REDIS_VERSION=6.2.1
+RUN curl -OL https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
+    tar xzf redis-${REDIS_VERSION}.tar.gz && \
+    cd redis-${REDIS_VERSION} && \
+    make && \
+    cp ./src/redis-server /usr/local/bin/ && \
```
Comment (k2n) [inline Dockerfile:45]: No `make install` target?

Comment (kgann) [reply Dockerfile:45]: There is but the default is to `install`. I'll update just to make it clear.

```
kgann:redis-6.2.1 ⇒ pwd
/Users/kgann/Downloads/redis-6.2.1
kgann:redis-6.2.1 ⇒ cat Makefile
# Top level makefile, the real shit is at src/Makefile

default: all

.DEFAULT:
	cd src && $(MAKE) $@

install:
	cd src && $(MAKE) $@

.PHONY: install
```

**Pull request merged**